tools/install/windows/merge/BrokerMerge.wxs

```diff
@@ -26,8 +27,9 @@
                     Profile="all" Protocol="udp" Scope="any" IgnoreFailure="no" />
               </File>
               <File Id="DnsSdDll" Name="dnssd.dll" DiskId="1" />
-              <ServiceControl Id="BrokerServiceCtl" Name="ETC RDMnet Broker" Stop="both"
-                  Start="install" />
+
+              <!-- Configures the actual installing and uninstalling of the service -->
+              <?include GeneratedFiles\service_config.wxi ?>
```

Comment:
Was having cmake generate a file with the values required a cleaner solution than just having cmake pass these in as variables to the wix compiler as cmd line arguments?

Reply:
I would say it depends on the number of arguments we would have to pass.  One or two is fine, but more than that a separate include is easier to maintain.

Reply:
Agree with @nwagner , have a look at src/windows/service_config.wxi.in; every value enclosed in "@" symbols would need to be passed at the command line. Plus, in my experience it's easier to reason about software if all or at least most of your definitions are in a source file -- hunting into the build tools and compiler invocation commands to find where something is defined makes the software more difficult to understand. Using the file is cleaner IMO.

Reply:
That makes perfect reasonable sense. Just didn't know if there was a reason beyond ease of maintenance.